rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/10_settings.yml

```diff
@@ -230,3 +252,43 @@ set start_time and end_time without timeseries mode:
             index:
               time_series:
                 end_time: 1632625782000
+
+---
+set bad start_time and end_time:
+  - skip:
+      version: " - 8.0.99"
+      reason: introduced in 8.1.0
+  - do:
+      indices.create:
+        index: test_index
+        body:
+          settings:
+            index:
+              mode: time_series
+              routing_path: [ metricset ]
+              time_series:
+                start_time: 1632625782000
+                end_time: 1632625792000
+          mappings:
+            properties:
+              metricset:
+                type: keyword
+                time_series_dimension: true
+
+  - do:
+      catch: /time series index @timestamp value \[1632625781000\] must be larger than 1632625782000/
+      index:
+        index: test_index
+        body: {
+          "@timestamp": 1632625781000,
+          "metricset": "pod"
+        }
+
+  - do:
+      catch: /time series index @timestamp value \[1632625793000\] must be smaller than 1632625792000/
+      index:
+        index: test_index
+        body: {
+          "@timestamp": 1632625793000,
+          "metricset": "pod"
```

Comment:
I think #80932 will have these too, right?

Reply:
yeah, when merged  #80932, this will change the exception info